suite-native/transactions/src/components/TransactionDetail/TransactionDetailSheets.tsx

```diff
@@ -25,6 +26,9 @@ export const TransactionDetailSheets = ({
     const toggleSheet = (sheetName: SheetType) => {
         setExpandedSheet(expandedSheet === sheetName ? null : sheetName);
     };
+
+    const valuesSheetIsVisible = !isTestnet(transaction.symbol);
```

Comment:
`isValuesSheetVisible` would be a better name but consider a better way to handle it as mentioned in previous comments

Reply:
fixed in 81e3b1287